napari/utils/events/event.py

```diff
@@ -71,6 +72,19 @@
 
 from ..translations import trans
 
+try:
+    # this could move somewhere higher up... but where?
+    __import__('dotenv').load_dotenv()
+except ImportError:
+    pass
+
+if os.getenv("NAPARI_DEBUG_EVENTS", '').lower() in ('1', 'true'):
+    from .debugging import log_event_stack
+else:
+
+    def log_event_stack(event):  # type: ignore
+        pass
```

Comment:
This happens at startup.  I didn't see a way to enable event tracing during runtime.  It might be nice to be able to turn the tracing on/off just during periods of interest.

Reply:
k, will think about that

Reply:
updated